docker/README.md

```diff
@@ -157,7 +157,7 @@ bob$ lncli listpeers
 Create the `Alice<->Bob` channel.
 ```bash
 # Open the channel with "Bob":
-alice$ lncli openchannel --node_key=<bob_identity_pubkey> --num_confs=1 --local_amt=1000000
+alice$ lncli openchannel --peer_id=1 --local_amt=1000000
```

Comment:
The `peer_id` arguments are largely being phased out, why the change to identitfy by the peer ID? As is now, users won't be notified of the peer ID of a peer upon connection.